Create yant.csv

kintaro1981 opened the pull request:
https://github.com/maxcleme/beadcolors/issues/48

maxcleme commented on raw/yant.csv:
```diff
@@ -0,0 +1,119 @@
+#02,white,255,255,255,KintaroOe
```
Do you mind dropping that `#` or maybe using `Y` for "yant" maybe?

```suggestion
02,white,255,255,255,KintaroOe
```

```suggestion
Y02,white,255,255,255,KintaroOe
```

kintaro1981 replied:
> Do you mind dropping that `#` or maybe using `Y` for "yant" maybe?

I simply used their numbering mode. I think there is no problem dropping the # or substitute it with the Y.

maxcleme reviewed (commented):
Did you exclude P and L beads on purpose?

kintaro1981 commented:
> Did you exclude P and L beads on purpose?

yes, but I forgot to creat separated csv.
Let me know if it's preferable to include them in the "main" csv o create separated files

maxcleme commented:
> > Did you exclude P and L beads on purpose?
> 
> yes, but I forgot to creat separated csv. Let me know if it's preferable to include them in the "main" csv o create separated files

I think we should include them in this CSV, yes

kintaro1981 commented:
- replaced # with Y
- added zeroes to have 3 digits reference codes

I double checked and I noticed that RGB values for P (Pearl) and L (glow in the dark) are not correct.
I think it's better to avoid them for now.

maxcleme commented:
Dang, we forgot to update the README! 🙈 

Do you mind opening another PR when you'll have time? 🙇
